query-service/src/integrationTest/java/org/hypertrace/core/query/service/htqueries/HTPinotQueriesTest.java

```diff
@@ -291,6 +291,21 @@ public void testServicesQueries() {
     assertTrue(serviceNames.isEmpty());
   }
 
+  @Test
+  public void testServicesQueriesForAvgRate() {
+    LOG.info("Services queries for AVG_RATE");
+    Iterator<ResultSetChunk> itr = queryServiceClient.executeQuery(
+        ServicesQueries.buildAvgRateQuery(), TENANT_ID_MAP, 10000);
+    List<ResultSetChunk> list = Streams.stream(itr).collect(Collectors.toList());
+    List<Row> rows = list.get(0).getRowList();
+    assertEquals(4, rows.size());
+    List<String> serviceNames = new ArrayList<>(
+        Arrays.asList("frontend", "driver", "route", "customer"));
+    rows.forEach(row -> serviceNames.remove(row.getColumn(1).getString()));
```

Comment:
I am curious about what `1` stands for here? Shall we make it a constant like `NAME_COLUMN` or something like that?

Reply:
Column indexed at 1 stores column names of the returned rows. Can be renamed to some constant variable indicating which column though.